hwi/oauth-bundle/0.6/post-install.txt

```diff
@@ -0,0 +1,20 @@
+<bg=blue;fg=white>                                          </>
+<bg=blue;fg=white> What's next for HWIOAuthBundle? </>
+<bg=blue;fg=white>                                          </>
+
+  * <fg=blue>Configure</> your application:
+    1. Enable sessions by uncommenting the entire session section in <comment>config/packages/framework.yaml</>
```

Comment:
This instruction is not really reliable. Maybe write exactly what to do here or refer to documentation. 

> Enable sessions by adding `framework.session.enabled: true`  in <comment>config/packages/framework.yaml